### ISSUE TITLE
Zoom level keeps getting set to zoomed out

### ISSUE THREAD

**Issue by Charles-Gagnon** [microsoft/azuredatastudio#15495]
RC1/insiders

I've seen this a fair amount but haven't been able to figure out an exact cause for why this might be happening. But every once in a while when bringing the ADS window to the front the zoom level of the app keeps getting set to zoomed out (so everything is much smaller). Resetting the zoom level brings it back to the default for a bit but then eventually it'll get set to being zoomed out again.

**Comment by Charles-Gagnon** [comment]
Might be related to https://github.com/microsoft/azuredatastudio/issues/14128

I started investigating that a bit to see if there was anything obvious going on there. What I found out was that in that case somehow a window resize event is being fired, which causes https://github.com/microsoft/azuredatastudio/blob/main/src/vs/workbench/browser/layout.ts#L1194 to resize the container based on the client area and ending up with the same result as this issue. I dug into it a bit but couldn't find out what was firing this event and why it wasn't taking the zoom level into account properly when updating the size - that will need some more investigation.
